.Rbuildignore

```diff
@@ -43,4 +43,4 @@
 ^data-raw$
 ^LICENSE\.md$
 ^docker\.sh$
-^dry$
+^deduplicate$
```

Comment:
DRY is short for Don't Repeat Yourself (https://en.wikipedia.org/wiki/Don%27t_repeat_yourself). 
I removed the jargon, favoring a more explicit name -- thanks CJ.

Reply:
hahahaha… awesome, learn something new every day 😉